composer.json

```diff
@@ -76,7 +76,7 @@
         }
     ],
     "require-dev": {
-        "appwrite/sdk-generator": "0.19.5",
+        "appwrite/sdk-generator": "dev-feat-new-headers",
```

Comment:
Reminder: Update this version.

Reply:
We need to release a new SDK generator version first 👍